### ISSUE TITLE
Pandoc doesn't create Allfiles. Error in script

### ISSUE THREAD

**Issue by travellingtrainer** [MicrosoftLearning/10979-MicrosoftAzureFundamentals#65]
Says No Allfiles folder found in working directory. Creating the folder doesn't help.
Creates Instructions correctly

**Comment by polichtm** [comment]
You can ignore the error. There is no content in Allfiles in this course. I created a placeholder.txt file in Allfiles to supress the error.
rgds
Marcin

**Comment by travellingtrainer** [comment]
Ok. Thanks. Are we to continue with the D version VM, as MS hasn't put an E version on the class download site.

**Comment by trayhumphrey** [comment]
I am currently teaching the E version of the book and labs on the D image. All is working fine. If you look at the Change Log for the E version, you'll see the image didn't really change.

**Comment by polichtm** [comment]
Appreciate the clarification
rgds
Marcin

**Comment by tbkobberoe** [comment]
@trayhumphrey @polichtm Someone should tell the moderators on the MCT support forum [https://trainingsupport.microsoft.com/en-us/tcmct/forum?sort=LastReplyDate&dir=Desc&tab=All&status=all&mod=&modAge=&advFil=&postedAfter=&postedBefore=&threadType=All&isFilterExpanded=false&page=1](url) about the fact that you are supposed to run the E version on a D image, as no one seems to know. I would recommend though that you update SQL Server Management Studio to latest version, and that the students actually install the latest Azure Powershell module and version of Azure CLI, or run the labs from the Cloud Shell

**Comment by trayhumphrey** [comment]
@tbkobberoe Point taken. I'll mention it on the MCT support forum.

Here is what the E Change Log says about the VM:
"The VM configuration has not changed significantly from the course’s D version. Some of the tools that
are installed on the student computer have been updated to the latest version."

Since I taught the class two days after the E Skillpipe book was released, I assumed the image would be updated soon but hadn't been yet. My students had no trouble using the D image. PowerShell and CLI are only used for one module.

**Comment by tbkobberoe** [comment]
@trayhumphrey Good luck with the MCT support forum. I haven't had much luck getting anything solved there.
From what the Change log says it seems like there should be an E version of the VM, but apparently there's not. I have no idea who to call for that issue, but I'll follow your case if you raise one in the support forum :-)

**Comment by trayhumphrey** [comment]
@tbkobberoe Yep, agreed. Just posted to the MCT Courseware Forum: 10979E VM image.

**Comment by polichtm** [comment]
@tbkobberoe  and @trayhumphrey - appreciate the posting on the MCT forum - I'll follow up regarding availability of the E version of the VM. 
Aside from the newer versions of SSMS, Azure PowerShell, and CLI, pls let me know if you are seeing any issues with running the E version course on the D version of the image. This is obviously not ideal, but I'd want to make sure that any potential issues are remediated until the version E image becomes available
rgds
Marcin
